src/common/snippets/tests/src/utils/split_dim_m.cpp

```diff
@@ -16,7 +16,11 @@ std::string SplitDimensionMTest::getTestCaseName(testing::TestParamInfo<SplitDim
     const auto& input = obj.param.input;
     const auto& reference = obj.param.reference;
     std::ostringstream result;
-    result << "Batch=" << input.cur_batch << "_";
+    if (input.cur_batch.has_value()) {
+        result << "Batch=" << input.cur_batch.value() << "_";
+    } else {
+        result << "Batch=undefined_";
```

Comment:
"undefined" is a bit confusing: if the batch value is not set, 2D shape is used => there are just no batch dimension. Can we print smth like "no_batch_dim"?